### DISCUSSION TITLE
Issues using New-DbaComputerCertificate

### DISCUSSION THREAD

**Discussion by bjendro** [dataplat/dbatools#9321] [Q&A]
We are experiencing issues when trying to generate and push a certificate from our CA to a server.  We get the following errors repeatedly, and nothing we have tried has either changed the result or output.

We are looking for assistance here.

PS H:\> New-DbaComputerCertificate -CertificateTemplate "Web Server2" -KeyLength 2048 -CaServer CAcert -EnableException 
WARNING: [14:51:59][New-DbaComputerCertificate] Something went wrong
WARNING: [14:51:59][New-DbaComputerCertificate] Certificate Request Processor: The data is invalid. 0x8007000d (WIN32: 13 ERROR_INVALID_DATA)
C:\Users\donald\AppData\Local\Temp\5\server-db1.mydomain.local\request.inf([NewRequest] NotBefore = "4/5/2024")
WARNING: [14:51:59][New-DbaComputerCertificate] Certificate Request Processor: The system cannot find the file specified. 0x80070002 (WIN32: 2 ERROR_FILE_NOT_FOUND)
C:\Users\donald\AppData\Local\Temp\5\server-db1.mydomain.local\server-db1.mydomain.local.csr
Failure when attempting to create the cert on server-DB1. Exception:
At line:97891 char:9
+         throw $records[0]
+         ~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (server-DB1:String) [], Exception
    + FullyQualifiedErrorId : dbatools_New-DbaComputerCertificate

We have noticed when we run the command, it creates the request.inf file in the folder shown, but we never see a csr, specifically the one it shows above that it cannot find.  If we create a csr using this same tool, and put it in the above folder before running the command, it just deletes the csr and throws the same error.

**Comment by niphlod** [comment]
seems related to https://github.com/dataplat/dbatools/pull/9318 . Once it's merged and a new release comes out, the bug should be fixed.

**Reply by bjendro** [reply]
Thank you.  We downgraded dbatools to the previous version until this is resolved.
